### ISSUE TITLE
Root path for collections

### ISSUE THREAD

**Issue by RWOverdijk** [benwinding/react-admin-firebase#19]
I have firestore set up in such a way that different "organisations" have their own root path, so I can easily restrict their access to that specific collection.

Right now, using react-admin-firebase this works:

```tsx
<Resource options={{ label: 'Categories' }} name='organisation/BQ0S2IpuKE6YWmt8FwAe/categories' />
```

Which is okay, but looks weird. It'd be nice to be able to set a basePath for references. I'm thinking of something like this:

```tsx
const dataProvider = FirebaseDataProvider(config, { rootPath: 'organisation/BQ0S2IpuKE6YWmt8FwAe');

// ...

<Resource name='categories' />
```

Feel free to let me know if you think it's a dumb idea 😄 

**Update:** I think I should have called it a rootRef. Everything is a ref anyway.

**Comment by benwinding** [comment]
Hi Again @RWOverdijk,
That's a great idea actually! it would allow better segregation between parts of the database. This is definitely something I want to implement.
Cheers,
Ben

**Comment by benwinding** [comment]
Hi @RWOverdijk,
After a lot of testing this has finally been implemented in version `0.6.1`. Thanks for your patience and let me know if there's any issues.
Cheers,
Ben
